### PR TITLE
Paths

### PR DESCRIPTION
**Pull request by danhearn**
@lukeplowden I've changed some minor things in your work on the media player so the program can work on any computer using paths_lib. It's all working fine but could you check my just incase you think this could cause any issues? Thanks x x x

### CODE REVIEW

**Comment by danhearn** [comment]
Problems with serial paths and the ding model

**Review by lukeplowden — APPROVED** [review]
All good just removed one import from media_player.py

**Comment by lukeplowden** [comment]
Go ahead and merge <3

File: installation-application/media_player.py [hunk 1]
```diff
@@ -138,4 +137,4 @@ def terminate(self):
     VIDS_PATH = "./data/vids/"
     mp = MediaPlayer(GIFS_PATH, VIDS_PATH)
     mp.start_on_new_process()
-    mp.queue_media("vid-10")
\ No newline at end of file
+    mp.queue_media("vid-10")
```
Comment (danhearn) [inline installation-application/media_player.py:141]: is this meant to be there?

Comment (lukeplowden) [reply installation-application/media_player.py:141]: It's a half hearted test, when you run that file alone. Can be removed if you want

Comment (danhearn) [reply installation-application/media_player.py:141]: ok bbz, gotta fix some other stuff then will merge <3

**Pull request merged**